maltoolbox/language/classes_factory.py

```diff
@@ -12,8 +12,9 @@ class LanguageClassesFactory:
     def __init__(self, lang_graph):
         self.lang_graph = lang_graph
         self.json_schema = {}
+        self.__create_classes()
 
-    def generate_assets(self):
+    def __generate_assets(self):
```

Comment:
A minor thing, prepending with 2 underscores will cause name mangling. This method will be accessible via:

```
class_factory_instance._LanguageClassesFactory__private_method()
```

Is there indeed a reason to prefer this over the single underscore convention for private methods?

Reply:
No, I just wasn't aware which of the two was preferred, I will change it in a future commit.